src/enums/privacyRegime.ts

```diff
@@ -13,5 +16,5 @@ export enum PrivacyRegimeEnum {
   /** Colorado Privacy Act */
   CPA = 'CPA',
   /** Nevada Senate Bill 220 */
-  'Nevada SB220' = 'Nevada SB220',
+  'NEVADA_SB220' = 'NEVADA_SB220',
```

Comment:
We should be able to use spaces and proper capitalization here. Is there really no other way?

Reply:
Can you explain why? It creates additional friction because we use the same enum in our GraphQL schema and spaces are not allowed there

Reply:
A customer looking at their regional experiences dashboard shouldn't be limited to all caps and no whitespace imo, so this will be pretty jarring to them imo

Reply:
@eligrey we can map this to a more human friendly string in dashboards/ui. This only impacts what gets stored in the database and what gets exposed via graphql

Reply:
This also affects customer code calling getRegimes(), though I do see your point. Can we at least use the normal case & spaces for the enum value (as opposed to the key)?